src/components/AssetHeader/QuickBuyEdit.tsx

```
@@ -0,0 +1,93 @@
+import { Button, Divider, FormControl, Input, Stack, useColorModeValue } from '@chakra-ui/react'
+import { isNaN } from 'lodash'
```

Biome (2.1.2)

[error] Do not shadow the global "isNaN" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)